geometry_msgs/Pose2D pose
uint8 period
uint8 duration
